src/components/MaplibreMap.jsx

```diff
@@ -268,13 +267,9 @@ const MaplibreMap = ({ view, setView }) => {
     }
   }
 
-  const MapAndTableControlsWrapper = () => {
-    const map = mapRef.current
-
-    if (map) {
-      return <MapAndTableControls map={map.getMap()} view={view} setView={setView} />
-    }
-  }
+  const mapAndTableControlsWrapper = mapRef.current ? (
+    <MapAndTableControls map={mapRef.current.getMap()} view={view} setView={setView} />
+  ) : null
```

Comment:
removing the nested component here is probably the only (maybe) performance improvement